packages/database/schema.prisma

```diff
@@ -166,14 +166,15 @@ enum DisplayStatus {
 }
 
 model Display {
-  id        String        @id @default(cuid())
-  createdAt DateTime      @default(now()) @map(name: "created_at")
-  updatedAt DateTime      @updatedAt @map(name: "updated_at")
-  survey    Survey        @relation(fields: [surveyId], references: [id], onDelete: Cascade)
-  surveyId  String
-  person    Person?       @relation(fields: [personId], references: [id], onDelete: Cascade)
-  personId  String?
-  status    DisplayStatus @default(seen)
+  id         String        @id @default(cuid())
+  createdAt  DateTime      @default(now()) @map(name: "created_at")
+  updatedAt  DateTime      @updatedAt @map(name: "updated_at")
+  survey     Survey        @relation(fields: [surveyId], references: [id], onDelete: Cascade)
+  surveyId   String
+  person     Person?       @relation(fields: [personId], references: [id], onDelete: Cascade)
+  personId   String?
+  responseId String?       @unique
+  status     DisplayStatus @default(seen)
```

Comment:
Is this still needed in the new version since we now have the response Id?
Can we make it optional on the schema so we still have backward compatibility until everybody uses the new formbricks-js with new version and then remove it?